### ISSUE TITLE
ON HOLD: PDP Page Overlay : View All Images with pagination

### ISSUE THREAD

**Issue by Meenu-GSPANN** [aemsites/danaher-abcam#18]
![image](https://github.com/user-attachments/assets/fd53a01d-3225-479b-9dc1-11ec6a8d5195)

after clicking on Images- View All, below is the sample overlay link:
https://www.abcam.com/en-in/products/primary-antibodies/vimentin-antibody-epr3776-cytoskeleton-marker-ab92547#overlay=images

**Comment by amlan18041996** [comment]
### [ 16-08-2024 ]:-
**Implemented:**

- Backdrop with configuration added to drawer script
- Paginated images
- Carousel implemented

**Issues:** Navigations are not working as expected

**In-Progress:**

- Navigations section of the image's carousel
- On-Select image along with its distinct description
- Responsiveness

**Comment by amlan18041996** [comment]
### **[ 20-08-2024 ]:-**

**Implemented:**
- Resolved carousel navigation
- On-select feature
- Responsiveness ( Mobile, iPad, Desktop )

**Issues:** Not able to receive the description along with their images _( Needs to be discussed )_

**Comment by amlan18041996** [comment]
### [ 04-09-2024 ]:-

- Will start implementation once design is complete. Though, the structural implementation is already completed on optimus repository.

**Comment by Meenu-GSPANN** [comment]
Closing the issue due to the design changed
